admin-scripts/addLastEdited.py

```diff
@@ -0,0 +1,19 @@
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import db
+import datetime
+from firebase_admin import firestore
+
+cred = credentials.Certificate('firebaseadmin.json')
+firebase_admin.initialize_app(cred)
+
+db = firestore.client()
+tasks = db.collection(u'samwise-tasks').get()
+
```

Comment:
@jessicahong9 could you change the code so that it uses `serverTimestamp`? Also, could you also add the new field for `tags` and `settings`.